src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +103,53 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={query}
+              onChange={(e) => onQueryChange(e.target.value)}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          { isError && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              disabled={isDisabled}
+              className={`button is-light${isLoading ? ' is-loading' : ''}`}
             >
-              Find a movie
+              {isSearched || 'Find a movie'}
```

Comment:
```suggestion
              {!isSearched && 'Find a movie'}
```
[conditional rendering](https://react.dev/learn/conditional-rendering)

the same below